.dockerignore

```diff
@@ -0,0 +1,4 @@
+build/*
```

Comment:
I _think_ this wouldn't ignore anything in sub-directories (so better not to have `*`).

Reply:
(I know the syntax is different to `.gitignore`; not actually tried so could be wrong.)